docs/network/edges.html

```diff
@@ -324,6 +325,12 @@ <h3>Options</h3>
             <td><code>Object</code></td>
             <td>Exactly the same as the to object but with an arrowhead at the from node of the edge.</td>
         </tr>
+        <tr>
+            <td class="indent">endpointsAdjustedPosition</td>
+            <td>Number</td>
+            <td><code>0</code></td>
+            <td>Set a number to adjust the endpoints. It can be useful if you want to have the endpoints placed on the nodes border. Negative numbers move the endpoints nearer the nodes midpoint, and positive numbers does the opposite. It's only possible when the arrowStrikethrough is true.</td>
+        </tr>
```

Comment:
We have Korean docs. Could you add it there too, please?

PS: You can replace Korean by TODO if you don't speak Korean. Someone will hopefully translate it later.

Reply:
Sorry, I missed that. Of course I will add TODO notes.